schema-lending.graphql

```diff
@@ -82,6 +91,17 @@ enum LendingType {
   POOLED
 }
 
+enum PermissionType {
+  " Protocols that only allow KYC'd users "
+  PERMISSIONED
+
+  " Protocols that allow a mix of KYC'd and non-KYC'd users "
+  PARTIALLY_PERMISSIONED
+
+  " Protocols that do not KYC. Can be used by any account "
+  PERMISSIONLESS
+}
+
 enum RiskType {
```

Comment:
Should we add something like a "Collateralization Type" to describe Over-collateralized, Under-collateralized, and uncollateralized lending protocol types?